packages/suite-data/package.json

```diff
@@ -9,24 +9,18 @@
     "license": "SEE LICENSE IN LICENSE.md",
     "description": "Trezor data",
     "scripts": {
-        "build": "yarn build:browser-detection",
-        "build:browser-detection": "webpack --config ./browser-detection.webpack.js",
-        "copy-static-files": "yarn build && ts-node ./index.ts",
+        "copy-static-files": "ts-node ./index.ts",
         "test-same-svgs": "ts-node ./test/test-same-svgs.ts"
     },
-    "dependencies": {
-        "ua-parser-js": "^0.7.24"
```

Comment:
i'm not sure we are able to touch server configs.
suite-web is running on some amazon aws.
second thing is that it will require additional maintenance by admins (sldev) and it will not work on localhost

Reply:
Browser detect has been reverted (c82d5f6).